docs/led-array-controller.md

```diff
@@ -31,20 +31,20 @@ node eg/led-array-controller.js
 
 
 ```javascript
-var five = require("johnny-five");
-var board = new five.Board();
+const {Board, Leds, Sensor} = require("johnny-five");
+const board = new Board();
 
 board.on("ready", function() {
-  var leds = new five.Leds([2, 3, 4, 5, 6]);
-  var pot = new five.Sensor("A0");
+  const leds = new Leds([2, 3, 4, 5, 6]);
+  const pot = new Sensor("A0");
 
-  pot.scale([-1, 4]).on("change", function() {
-    var lastIndex = Math.round(this.value);
+  pot.scale([-1, 4]).on("change", () => {
+    const lastIndex = Math.round(pot.value);
```

Comment:
All uses of `sensor.scale(...)` should be removed and the code updated to use the `scaleTo()` method. So for example, this code would become: 

```js
pot.on("change", () => {
   const lastIndex = Math.round(pot.scaleTo(-1, 4));
   // ...
```